src/components/Claim/CreateSource/SourceEditor.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useCallback } from "react";
+import { PlaceholderExtension } from "remirror/extensions";
+import { Remirror, useRemirror } from "@remirror/react";
+import { TrailingNodeExtension } from "remirror/extensions";
+import SummaryExtension from "../../Collaborative/Form/SummaryExtesion";
+import SourceExtension from "../../Collaborative/Form/SourceExtesion";
+import ProsemirrorEditorStyle from "../../Collaborative/ProsemirrorEditor.style";
+import SourceEditorButton from "./SourceEditorButton";
+
+const SourceEditor = ({ onContentChange }) => {
+    function createExtensions() {
```

Comment:
thought: in the future version of the editor, extensions should be configurable.